docs/application/dotnet/guides/connectivity/webview.md

```diff
@@ -0,0 +1,165 @@
+
+    You can [load a web page](#load) with requested URL and [register event handlers](#load) to load events.
+
+- Managing Cookie
+
+    You can [set the cookie policy and the storage](#cookie) for the cookie.
+
+- Finalizing WebView
+
+    You can [shut down the WebView](#finalize) and clean up the resources.
+
+## Prerequisites
+
+1. To use the [Tizen.WebView](https://samsung.github.io/TizenFX/latest/api/Tizen.WebView.html) namespace, create the `ElmSharp-Beta` project using the [Project Wizard](../../../vstools/tools/project-wizard.md) of Visual Studio.
+
+2. To use the class of the `Tizen.WebView` namespace, you must request for permission by adding the following privileges to the `tizen-manifest.xml` file, as shown in the following code snippet:
+
+   ```
+   <privileges>
+     <!--To use the Internet connection-->
+     <privilege>http://tizen.org/privilege/internet</privilege>
+
+     <!--To access media storage-->
+     <privilege>http://tizen.org/privilege/mediastorage</privilege>
+
+     <!--To access, read, and write to the external storage-->\
+     <privilege>http://tizen.org/privilege/externalstorage</privilege>
+   </privileges>
+   ```
+
+3. To use the methods and properties of the `Tizen.WebView` namespace, include it in your application, as shown in the following code snippet:
+    ```
+    using Tizen.WebView;
+    ```
+
+<a name="initialize"></a>
+## Initializing and Creating WebView
+
+To Initialize WebView and create a WebView object, follow these steps:
+
+1. To use the `Tizen.WebView` namespace in your application, call the `Tizen.WebView.Chromium.Initialize` method before the start of the main loop, as shown in the following code snippet:
+
+    ```
+    Elementary.Initialize();
+    Elementary.ThemeOverlay();
+    Chromium.Initialize();
+    App app = new App();
+    app.Run(args);
+    ```
+
+2. To create a WebView object, use the constructor of [Tizen.WebView.WebView](https://samsung.github.io/TizenFX/latest/api/Tizen.WebView.WebView.html) with `ElmSharp.Window`, as shown in the following code snippet:
+    ```
+    var webView = new WebView(window)
+    {
+	AlignmentX = -1,
+	AlignmentY = -1,
+	WeightX = 1,
+	WeightY = 1
+    };
+    webView.Show();
+    ```
+
+<a name="load"></a>
+## Managing Web Page Loading
+
+- To load a web page, use the `Tizen.WebView.WebView.LoadUrl` method with an appropriate URL, as shown in the following code snippet:
+
+    ```
+    webView.LoadUrl("https://www.tizen.org/");
+    ```
+
+- The following table lists the loading events provided by the `Tizen.WebView.WebView` class:
+
+**Table: Loading Events**
+
+| Event            | Description                                            | Argument                                 |
+|------------------|--------------------------------------------------------|------------------------------------------|
+| `LoadStarted`    | This event occurs when loading starts.                 | -                                        |
```

Comment:
@mijin85cho, @nawab-samsung 
This one is tricky actually.
Now that the developer has provided a clear and complete definition, how about re-writing it as:

This event occurs when a web page loading starts.
This event occurs when a web page loading completes.
This event occurs when a web page loading fails.

Reply:
On messenger, @annie-samsung and I fixed the suggested sentences as below.

 - This event occurs when the page loading has started.
 - This event occurs when the page loading has completed.
 - This event occurs when the page loading has failed with an error.

Reply:
@younghajung,
Kindly make the following changes:

    This event occurs when the page loading has started.
    This event occurs when the page loading has completed.
    This event occurs when the page loading has failed with an error.

Reply:
Done.